code/modules/modular_computers/file_system/program.dm

```diff
@@ -120,7 +120,7 @@
 		if(3)
 			ntnet_speed = NTNETSPEED_ETHERNET
 
-// Check if the user can run program. Only humans can operate computer. Automatically called in run_program()
```

Comment:
The proc is "can_run" but the comment is about downloading shit???

Reply:
fixed